conformance/lib/src/nativeMain/kotlin/pbandk/conformance/Platform.kt

```diff
@@ -29,30 +29,37 @@
 
 package pbandk.conformance
 
-import platform.posix.*
+import kotlinx.cinterop.ExperimentalForeignApi
 import kotlinx.cinterop.cstr
 import kotlinx.cinterop.usePinned
 import kotlinx.cinterop.addressOf
 import kotlinx.coroutines.CoroutineScope
 import kotlinx.coroutines.Dispatchers
-import kotlinx.coroutines.withContext
+import kotlinx.coroutines.IO
 import kotlinx.coroutines.runBlocking
+import kotlinx.coroutines.withContext
 import pbandk.Message
 import pbandk.decodeFromByteArray
 import pbandk.encodeToByteArray
+import platform.posix.posix_errno
+import platform.posix.read
+import platform.posix.strerror
+import platform.posix.write
 
+@OptIn(ExperimentalForeignApi::class)
 internal class PosixException(val errno: Int) : RuntimeException(
     strerror(errno)?.toString() ?: "Error with unknown errno: $errno"
 )
 
+@OptIn(ExperimentalForeignApi::class)
 actual object Platform {
     actual fun stderrPrintln(str: String) {
         val strn = str + "\n"
         val cstr = strn.cstr
         write(2, cstr, cstr.size.toULong())
     }
 
-    private suspend fun stdinReadIntLE() = withContext(Dispatchers.Main) {
+    private suspend fun stdinReadIntLE() = withContext(Dispatchers.IO) {
```

Comment:
Kotlin/Native now supports `Dispatchers.IO`, and `Dispatchers.Main` doesn't work correctly now that `Dispatchers.IO` is available. This brings Kotlin/Native more in line with how Kotlin/JVM works.